data-generation/src/main/scala/thesis/Landy.scala

```diff
@@ -31,16 +31,23 @@ class Landy(graph: LandyAttributeGraph) extends TemporalGraph[LandyEntityPayload
     )
     Graph(vertices, edges)
   }
-  
-  override def directNeighbours(vertexId: VertexId, interval: Interval): RDD[VertexId] = throw new NotImplementedError()
+
+  override def directNeighbours(vertexId: VertexId, interval: Interval): RDD[VertexId] = {
+    this.edges.filter(edge => {
+      val edgeInterval = Interval(edge.attr.validFrom, edge.attr.validTo)
+      edge.srcId == vertexId && interval.overlaps(edgeInterval)
```

Comment:
```suggestion
      (edge.srcId == vertexId || edge.dstId == vertexId) && interval.overlaps(edgeInterval)
```
Må antakeligvis ha noe sånt, for ved Edge(1L, 2L) så blir ikke 1 nabo til 2:)
Må også fikse mappen senere i funksjonen for å da hente ut den som er rett også, litt á la det jeg har gjort i #30